README.md

```diff
@@ -33,22 +33,26 @@
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
+- **[HTML]**
+- **[CSS]**
+- **[Javascript]**
 
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-- **[HTML]**
-- **[CSS]**
+- **[Full responsive]**
+- **[Interactive buttons]**
+- **[Navigational menu]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://bhekisile.github.io/Bee_Portfolio/)
+- [Live Demo Link](https://www.loom.com/share/00914bdf9755441fa29e4044c5ef647f)
```

Comment:
- [ ] Great effort working on the mobile menu of the portfolio project. Currently the link to your `live demo` is directing me to a loom recording which is not the deployed link of the project. Kindly add the correct link that you deployed